source/manual/deploying-terraform.html.md

```diff
@@ -57,15 +57,26 @@ Click "build" and terraform should deploy as expected. Remember to deploy to sta
 
 ### Other ways of invoking Jenkins
 
-As of version `v2.15.0` of `gds-cli`, you can use it to deploy terraform via Jenkins. Run `gds govuk terraform --help` for details.
-
-Alternatively, you can invoke this [script in the govuk-aws repository](https://github.com/alphagov/govuk-aws/blob/master/tools/deploy.rb):
+As of version `v2.15.0` of `gds-cli`, you can use it to deploy terraform via Jenkins.
```

Comment:
This caught me out:

```suggestion
As of version `v2.15.0` of `gds-cli`, you can use it to deploy terraform via Jenkins.
Note that you should use at least [v5.25.0](https://github.com/alphagov/gds-cli/releases/tag/v5.25.0) of `gds-cli`, otherwise you will need to manually specify the branches of `govuk-aws` and `govuk-aws-data` (see below).
```